pkg/langruntime/langruntime.go

```diff
@@ -309,6 +308,10 @@ func (l *Langruntimes) GetCompilationContainer(runtime, funcName string, install
 			"cp /kubeless/*.java /kubeless/function/src/main/java/io/kubeless/ && " +
 			"cp /kubeless/function-pom.xml /kubeless/function/pom.xml 2>/dev/null || true && " +
 			"mvn package > /dev/termination-log 2>&1 && mvn install > /dev/termination-log 2>&1"
+	case strings.Contains(runtime, "dotnetcore"):
+		command = "if [ -s /kubeless/project.csproj ]; then echo .csproj present; else cp /app/project.csproj /kubeless/project.csproj; fi && " +
```

Comment:
I would move this to a script in the image so this doesn't contain too much bash logic.

Reply:
I Agree. I will move this to a `compile.sh` script.